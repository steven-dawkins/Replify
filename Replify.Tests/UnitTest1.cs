using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using ApprovalTests.Reporters;
using System.Text;
using System.IO;
using ApprovalTests;

namespace Replify.Tests
{
    [UseReporter(typeof(DiffReporter), typeof(ClipboardReporter))]
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestHelp()
        {
            var output = new StringBuilder();
            var writer = new StringWriter(output);
            var repl = new ClearScriptRepl(writer);

            repl.Execute("help");

            Approvals.Verify(output);            
        }
    }
}
