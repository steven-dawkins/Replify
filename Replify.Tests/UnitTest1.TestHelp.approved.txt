Available commands:

Script

help
history
clearHistory
quit | exit
